### PR TITLE
Update module list

### PR DESCRIPTION
**Pull request by deepakazad**
Remove modules that no longer exist.
Add new modules.

### CODE REVIEW

File: conf-sample.edn [hunk 1]
```diff
@@ -17,11 +17,12 @@
  :resources {:group-id "com.elasticpath.rest.definitions"
              :format "%s/ep-resource-%s-api"
              :version "0-SNAPSHOT"
-             :modules #{addresses
-                        assets
+             :modules #{
```
Comment (arichiardi) [inline conf-sample.edn]: small nitpick, could you please remove the empty lines there? Aka data in contained inside the brackets. It reads more idiomatic :smile:

Comment (deepakazad) [reply conf-sample.edn]: Done :)

**Comment by arichiardi** [comment]
Thank you very much!

File: conf-sample.edn [hunk 2]
```diff
@@ -18,10 +18,10 @@
              :format "%s/ep-resource-%s-api"
              :version "0-SNAPSHOT"
              :modules #{addresses
-                        assets
                         availabilities
```
Comment (mattbishop) [inline conf-sample.edn:21]: Hmm, this list should be discovered from the classpath the same way the Java generator does, no? I can't see us asking customers to maintain this list for graphql when they don't for Helix.

Comment (arichiardi) [reply conf-sample.edn:21]: This conf is only needed for `boot` . For the maven plugin there is [full discoverability](https://github.com/elasticpath/rest-resource-viz/blob/301f973183fbdfa83326c1ce7c16253bdb8269b5/src/task/rest_resources_viz/extractor.clj#L363) and no list to maintain.

Comment (arichiardi) [reply conf-sample.edn:21]: Running from `boot` is more of a development workflow I would say...

**Pull request merged**